### PR TITLE
MINOR: Fix typos for connect

### PR DESCRIPTION
**Pull request by berg223**
*More detailed description of your change,
Fix typos for connect*

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: This change is incorrect; "pre-empted" is the intended term.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Also incorrect

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Also incorrect

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Also incorrect

**Review by C0urante — COMMENTED** [review]
Thanks @xiaocairush! LGTM except for the "pre-empted" changes. Once those are removed we can merge.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: resolved

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: resolved

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: Have reverted all false positive fix in WorkerConnector

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java
Comment (berg223) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java]: resolved

**Comment by C0urante** [comment]
Thanks! Will merge pending CI build.

**Review by C0urante — APPROVED** [review]

**Pull request merged**
